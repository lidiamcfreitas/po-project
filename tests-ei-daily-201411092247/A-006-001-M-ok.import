USER|skywalker|Luke Skywalker|/home/skywalker
USER|sidious|Darth Sidious|/home/sidious
USER|vader|Darth Vader|/home/vader
USER|yoda|Master Yoda|/home/yoda
DIRECTORY|/home/yoda/droids|yoda|public
DIRECTORY|/home/vader/friends|vader|public
