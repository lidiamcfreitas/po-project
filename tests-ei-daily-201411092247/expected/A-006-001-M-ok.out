Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Login
5 - Menu Shell
6 - Menu Utilizador
0 - Sair
Escolha uma opção: Menu Utilizador
1 - Criar utilizador
2 - Listar utilizadores
0 - Sair
Escolha uma opção: root:super user:/home/root
sidious:Darth Sidious:/home/sidious
skywalker:Luke Skywalker:/home/skywalker
vader:Darth Vader:/home/vader
yoda:Master Yoda:/home/yoda
Menu Utilizador
1 - Criar utilizador
2 - Listar utilizadores
0 - Sair
Escolha uma opção: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Login
5 - Menu Shell
6 - Menu Utilizador
0 - Sair
Escolha uma opção: 